app/main.py

```diff
@@ -1,3 +1,14 @@
 def cache(func):
-    # Write your code here
-    pass
+    cache_of_args = {}
+
+    def wrapper(*args, **kwargs):
+        i = args
```

Comment:
you don't need `i` variable, it's unnecessary in this case

Reply:
ok, going to change